fenix/app/src/main/res/values/strings.xml

```diff
@@ -290,8 +290,9 @@
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
     <string name="juno_onboarding_default_browser_title" tools:ignore="UnusedResources">Make %s your go-to browser</string>
     <!-- Description for set firefox as default browser screen.
-        The first parameter is the string with key "juno_onboarding_default_browser_description_link_text", that says "privacy notice" in en-* -->
-    <string name="juno_onboarding_default_browser_description" tools:ignore="UnusedResources">Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our %s.</string>
+        The first parameter is the Firefox brand name.
+        The second parameter is the string with key "juno_onboarding_default_browser_description_link_text", that says "privacy notice", and that is possible to translate in another string -->
```

Comment:
```suggestion
        The second parameter is the string with key "juno_onboarding_default_browser_description_link_text" that says "privacy notice", and that is possible to translate in another string. -->
```
nit: punctuations

I am wondering where you say "that is possible to translate in another string" you mean that %2$s could have a different string key that isn't "juno_onboarding_default_browser_description_link_text"? I think for translation purposes you want to kinda reassure them that the the string that is being filled here is predictive otherwise any changes to the this parameter from "privacy notice" might throw the translation off. 

I would consider figuring out if this is really gonna change and ideally dropping that possibility and obsoleting that bit of the localization comment.

Reply:
The intention was to say that %2$s will be the string with key `juno_onboarding_default_browser_description_link_text`.  I was pointed out [here](https://github.com/mozilla-l10n/android-l10n/pull/634#discussion_r1146968846) to add this so I went with it 😅

Reply:
Okay maybe double check with @Delphine before landing.

Reply:
Thanks for looping me in!
With a fresh brain, I agree the second part of the comment I made ("and that is possible to translate in another string.") can be confusing, and we could leave it out to make things clearer - since %2$s can only be the string with key `juno_onboarding_default_browser_description_link_text`

Reply:
great, thank you both!